libndt.hpp

```diff
@@ -1960,6 +2017,20 @@ bool Client::ndt7_upload() noexcept {
     constexpr auto measurement_interval = 0.25;
     std::chrono::duration<double> interval = now - latest;
     if (interval.count() > measurement_interval) {
+#ifdef __linux__
+      // Read tcp_info data for the socket and print it as JSON.
+      struct tcp_info tcpinfo{};
+      socklen_t soerrlen = sizeof(tcpinfo);
+      if (sys_getsockopt(sock_, IPPROTO_TCP, TCP_INFO, (void *)&tcpinfo,
+                             &soerrlen) == 0) {
```

Comment:
Please align `&soerrlen` with `sock_`

Reply:
I've run `clang-format` which had a few ideas on how to improve `libndt.hpp`. The changes look reasonable to me - what do you think?